### PR TITLE
DOCSP-27817 manual tagging for docs/ecosystem pages

### PR DESCRIPTION
**Pull request by sarahemlin**
Hey @caitlindavey & @mongoKart -

Here are the manual taxonomy tags for the docs-ecosystem pages from the top 250.  Please merge if everything looks good to you! 

Thanks!
Sarah

JIRA - <https://jira.mongodb.org/browse/DOCSP-27817>

### CODE REVIEW

File: source/cxx.txt [hunk 1]
```diff
@@ -4,6 +4,10 @@
 MongoDB C++ Driver
 ==================
 
+.. facet::
+   :type: programming_language
+   :values: csharp, dotnet
```
Comment (mongoKart) [inline source/cxx.txt:9]: I: This should be marked "cpp", not csharp or dotnet

File: source/index.txt [hunk 2]
```diff
@@ -6,6 +6,9 @@
 Start Developing with MongoDB
 =============================
 
+.. meta:: 
+   :keywords: r, elixir, prisma, mongoose
```
Comment (mongoKart) [inline source/index.txt:10]: Q: Should we include all languages in this list?

File: source/java-drivers.txt [hunk 3]
```diff
@@ -6,6 +6,17 @@ MongoDB Java Drivers
...
+.. facet::
+  :type: programming_language
+  :values: java
+
+.. facet::
+  :type: genre
+  :values: reference
+
+.. meta:: 
+  :keywords: java sync, java async
```
Comment (mongoKart) [inline source/java-drivers.txt:18]: S: The async driver is also called "reactive streams"--probably good to add here

File: source/php.txt [hunk 4]
```diff
@@ -6,6 +6,17 @@ MongoDB PHP Driver
 
 .. default-domain:: mongodb
 
+.. facet::
+   :type: programming_language
+   :values: php
+
+.. facet::
+   :type: genre
+   :values: tutorial
```
Comment (mongoKart) [inline source/php.txt:15]: I: Probably closer to reference than tutorial

File: source/pymongo.txt [hunk 5]
```diff
@@ -2,6 +2,17 @@
 PyMongo
 =======
 
+.. facet::
+   :type: programming_language
+   :values: python
+
+.. facet::
+   :type: genre
+   :values: tutorial
```
Comment (mongoKart) [inline source/pymongo.txt:11]: Reference, not tutorial

File: source/index.txt (hunk 2 above)
Comment (sarahemlin) [reply source/index.txt:10]: That wasn't part of the approved tagging list for this round, but getting the landing page totally tagged is top of my list for Munich.

File: source/java-drivers.txt (hunk 3 above)
Comment (sarahemlin) [reply source/java-drivers.txt:18]: Keywords are flexible, so I'll add this one.  Do you think that reactive streams is better than java async, or do users use both terms?

File: source/pymongo.txt (hunk 5 above)
Comment (sarahemlin) [reply source/pymongo.txt:11]: approved by Docs Leads

File: source/cxx.txt (hunk 1 above)
Comment (sarahemlin) [reply source/cxx.txt:9]: Thank you; this file isn't one of our top 250, so I'll delete this one and add csharp.  My notes had mistakenly said that the csharp landing page was in docs-ecosystem, but it's in docs-csharp.

**Review by sarahemlin — COMMENTED** [review]
@mongoKart I think we're good to go now on this one after I deleted the cxx file.

**Review by sarahemlin — COMMENTED** [review]
@mongoKart really done now. ;)
cxx file deleted & tag added for reactive streams.  Thank you!

**Review by mongoKart — APPROVED** [review]
lgtm

**Pull request merged**